src/components/styles/GlobalStyles.tsx

```diff
@@ -28,11 +29,17 @@ const imagery = css`
   }
 `;
 
+const tooltipGlobalStyles = css`
+  @import url('https://unpkg.com/tippy.js@${tippyVersion}/dist/tippy.css');
+  @import url('https://unpkg.com/tippy.js@${tippyVersion}/animations/shift-toward.css');
+`;
+
```

Comment:
Adding it in `<GlobalStyles />` like this is included and run once in the client `<App />` component.

If I add it as part of `<Tooltip />` it might get run multiple times ( due to multiple instances of `<Tooltip />` and re-renders ) and cause strange side effects... 💆🏻‍♂️